### ISSUE TITLE
Swipeable Component does not work on RELOAD

### ISSUE THREAD

**Issue by jacargentina** [software-mansion/react-native-gesture-handler#1964]
Just trying to migrate my app to RN 0.68.0-rc.4

My JS code is the very same, but I'm trying to use the new architecture like this

`RCT_NEW_ARCH_ENABLED=1 pod install`
`yarn ios`

My Swipeable rows not working anymore.

**Comment by jacargentina** [comment]
# Repo for reproduction

https://github.com/jacargentina/react-native-gesture-handler/tree/main

```
yarn install
cd FabricExample
yarn install
cd ios
RCT_NEW_ARCH_ENABLED=1 pod install
cd ..
yarn ios
```

Then swipe **Swipe me to left**. Works.

then press R to reload

Try to swipe **Swipe me to left** again. It will not work.

**Comment by jacargentina** [comment]
BTW on my real app, I don't even make any reload.

**Comment by tomekzaw** [comment]
Hello! @jacargentina Indeed, gestures are no longer working after reload on iOS. The good news is that we know the root cause. We are working on a fix and hope to deliver it with 2.4.0 release soon.

**Comment by jacargentina** [comment]
@tomekzaw is this fixed on 2.4.0?

**Comment by tomekzaw** [comment]
@jacargentina Yes, this should be fixed with #1988 which was included in the 2.4.0 release.

**Comment by jacargentina** [comment]
Works! 👌🏼
